azure-communication-ui/azure-communication-ui-demo-app/src/main/java/com/azure/android/communication/ui/callingcompositedemoapp/features/SettingsFeatures.kt

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.ui.callingcompositedemoapp.features
+
+import android.content.Context
+import com.azure.android.communication.ui.callingcompositedemoapp.DEFAULT_LANGUAGE_VALUE
+import com.azure.android.communication.ui.callingcompositedemoapp.DEFAULT_RTL_VALUE
+import com.azure.android.communication.ui.callingcompositedemoapp.LANGUAGE_ADAPTER_VALUE_SHARED_PREF_KEY
+import com.azure.android.communication.ui.callingcompositedemoapp.LANGUAGE_CUSTOM_TRANSLATION_ENABLE
+import com.azure.android.communication.ui.callingcompositedemoapp.LANGUAGE_ISRTL_VALUE_SHARED_PREF_KEY
+import com.azure.android.communication.ui.utilities.implementation.FEATURE_FLAG_SHARED_PREFS_KEY
+
+class SettingsFeatures {
+
+    companion object {
+        var isLanguageFeatureEnabled: Boolean = false
```

Comment:
Can use inference in kotlin
var isLanguageFeatureEnabled = false

No need to put type.